py/sentry_streams/user_functions/agg_template.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC, abstractmethod
+from typing import Any
+
+
+class Accumulator(ABC):
+
+    @abstractmethod
+    def create(self) -> Any:
+        raise NotImplementedError
+
+    @abstractmethod
+    def add(self, acc: Any, value: Any) -> Any:
+        raise NotImplementedError
+
+    @abstractmethod
+    def get_output(self, acc: Any) -> Any:
+        raise NotImplementedError
+
+    @abstractmethod
+    def merge(self, acc1: Any, acc2: Any) -> Any:
+        raise NotImplementedError
```

Comment:
Hint. Try to make this generic rather than have `Any` as type everywhere.

Reply:
will do, right now just getting things working with Flink